scripts/convert-pfx-cert-format.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -ex
+
+usage()
+{
+    echo "Usage:  $0 -p <pfx_cert> -k <signing_key> -c <signing_cert> -a <ca_cert>"
+    exit 1
+}
+
+while getopts "p:k:c:a:" opt; do
+    case $opt in
+        p)
+            PFX_FILE=$OPTARG
+            ;;
+        k)
+            SIGNING_KEY=$OPTARG
+            ;;
+        c)
+            SIGNING_CERT=$OPTARG
+            ;;
+        a)
+            CA_CERT=$OPTARG
+            ;;
+        *)
+            usage
+            ;;
+    esac
+done
+
+
+( [ -z $PFX_FILE ] || [ -z $SIGNING_KEY ] || [ -z $SIGNING_CERT ] || [ -z $CA_CERT ] ) && exit 1
+
+openssl pkcs12 -in "${PFX_FILE}" -clcerts -nokeys -nodes -passin pass: | sed -z -e "s/.*\(-----BEGIN CERTIFICATE\)/\1/" > ${SIGNING_CERT}
```

Comment:
Instead of copying the contents over to a file, you can specify the output file name as an input to the openssl command. 
For ex: openssl pkcs12 -clcerts -nokeys -in <cert>.pfx -out <output_cert>.crt -password pass: -passin pass:
Any specific reason why you choose this approach?

Reply:
Only because there are some additional attributes in the output files.
I think you are right, the attributes should not have any impacts when it is used to sign the swi image, I will test it later. 
Thanks for the great comment.
**Bag Attributes: <No Attributes>
subject=C = US, ST = Test, L = Test, O = Test, CN = Test

issuer=C = US, ST = Test, L = Test, O = Test, CN = Test**

-----BEGIN CERTIFICATE-----
MIIFdTCCA12gAwIBAgIUL2kglpzjw8n7sLr41bLDrLU8CcswDQYJKoZIhvcNAQEL
BQAwSTELMAkGA1UEBhMCVVMxDTALBgNVBAgMBFRlc3QxDTALBgNVBAcMBFRlc3Qx
DTALBgNVBAoMBFRlc3QxDTALBgNVBAMMBFRlc3QwIBcNMjAwNjI5MDYyNzE4WhgP